sw/device/sca/ecc384_serial.c

```diff
@@ -113,10 +131,10 @@ static const otbn_addr_t kOtbnVarK = OTBN_ADDR_T_INIT(p384_ecdsa_sca, k);
 /**
  * Makes a single dptr in the P384 library point to where its value is stored.
  */
-static void setup_data_pointer(otbn_t *otbn_ctx, const otbn_addr_t dptr,
+static void setup_data_pointer(const otbn_addr_t dptr,
                                const otbn_addr_t value) {
-  SS_CHECK(otbn_copy_data_to_otbn(otbn_ctx, sizeof(value), &value, dptr) ==
-           kOtbnOk);
+  SS_CHECK(otbn_dmem_write(sizeof(value) / sizeof(uint32_t), &value, dptr) ==
+           kOtbnErrorOk);
```

Comment:
That link is for the `silicon_creator` OTBN driver; this code is using the [cryptolib driver version](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/crypto/drivers/otbn.h;l=251). Normally these two drivers match; the skew right now is because of an in-progress OTBN driver refactor (#16789). Currently the cryptolib one has been updated but the `silicon_creator` one hasn't yet.